README.md

```diff
@@ -56,154 +73,145 @@ You can also tab complete remote names and branch names e.g.: `git pull or<tab>
-   If you have Git installed, make sure the path to git.exe is in your PATH environment variable.
+   If you have Git installed, make sure the path to git is in your PATH environment variable.
 
-### Installing posh-git via PowerShellGet
-If you are on PowerShell version 5 or higher, execute the command below to install from the [PowerShell Gallery](https://www.powershellgallery.com/):
+### Installing posh-git
 
-```powershell
-PowerShellGet\Install-Module posh-git -Scope CurrentUser
-```
-You may be asked if you trust packages coming from the PowerShell Gallery. Answer yes to allow installation of this module to proceed.
+posh-git is available on the [PowerShell Gallery][psgallery-beta1] and can be installed using the PowerShellGet module.
 
-If you are on PowerShell version 3 or 4, you will need to install the [Package Management Preview for PowerShell 3 & 4](https://www.microsoft.com/en-us/download/details.aspx?id=51451) in order to run the command above.
+1. Start either Windows PowerShell 5.x or PowerShell Core 6.x (`pwsh`).
 
-Note: If you get an error message from `Install-Module` about NuGet being required to interact with NuGet-based repositories, execute the following commands to bootstrap the NuGet provider:
-```powershell
-Install-PackageProvider NuGet -Force
-Import-PackageProvider NuGet -Force
-```
-Then retry the `Install-Module` command above.
+2. Execute one of the following two commands from an elevated PowerShell prompt depending on your current posh-git installation status:
 
-After you have successfully installed the posh-git module from the PowerShell Gallery, you will be able to update to a newer version by executing the command:
-```powershell
-Update-Module posh-git
-```
+   ```powershell
+   # If you have never installed posh-git from the PowerShell Gallery
+   # NOTE: If asked to trust packages from the PowerShell Gallery, answer yes to continue installation of posh-git
+   PowerShellGet\Install-Module posh-git -Scope CurrentUser -Force
+   ```
 
-### Installing posh-git via Chocolatey
-If you are having issues using `Install-Module` with PowerShell version 3 or 4, you can use [Chocolatey](https://chocolatey.org) to install posh-git.
-If you don't have Chocolatey, you can install it from the [Chocolately Install page](https://chocolatey.org/install).
-With Chocolatey installed, execute the following command to install posh-git:
-```powershell
-choco install poshgit
-```
+   OR
+
+   ```powershell
+   # If you have already installed a previous version of posh-git from the PowerShell Gallery
+   PowerShellGet\Update-Module posh-git
+   ```
+
+### Extra step for Linux and macOS users
+
+The version of `PSReadLine` (1.2) that ships with PowerShell Core 6.0 has known issues that cause problems with the
+prompt function that posh-git uses.  Fortunately, there is a beta of PSReadLine 2.0.0 that fixes these problems.
+You can check the version of PSReadLine you have by executing: `get-module psreadline`.
```

Comment:
Should we add a warning to `posh-git.psm1` if we detect this situation?

And super nitpicky, but maybe capitalize `Get-Module PSReadline`?